packages/react-core/src/components/NotificationBadge/NotificationBadge.tsx

```diff
@@ -27,6 +30,8 @@ export interface NotificationBadgeProps extends Omit<ButtonProps, 'variant'> {
   isExpanded?: boolean;
   /** Determines the variant of the notification badge. */
   variant?: NotificationBadgeVariant | 'read' | 'unread' | 'attention';
+  /** Determines if the notification badge should animate on new notifications. */
```

Comment:
```suggestion
  /** Flag indicating whether the notification badge animation should be triggered. */
```

Reply:
updated the description accordingly, thanks